src/views/TradingCompetition/components/YourScore/CardUserInfo.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import { Text, Heading, Flex, Skeleton } from '@pancakeswap-libs/uikit'
+import useI18n from 'hooks/useI18n'
+import { YourScoreProps } from '../../types'
+
+const CardUserInfo: React.FC<YourScoreProps> = ({ registered, account, profile }) => {
+  const TranslateString = useI18n()
+
+  const getHeadingText = () => {
+    if (!account) {
+      return 'Check your Rank'
+    }
+    if (!registered) {
+      return 'You’re not participating this time.'
+    }
+    if (profile) {
+      return `@${profile.username}`
+    }
+    return ''
+  }
+
+  const getSubHeadingText = () => {
+    if (!account) {
+      return 'Connect wallet to view'
+    }
+    if (!registered) {
+      return 'Sorry, you needed to register during the “entry” period!'
+    }
+    if (profile) {
+      return `${profile.team.name}`
```

Comment:
Same here

Reply:
Will do man - picking this work up again now